tests/lib/custom_lib/src/main.c

```diff
@@ -11,8 +11,8 @@
  * library works correctly.
  */
 
-#include <zephyr.h>
-#include <ztest.h>
+#include <zephyr/kernel.h>
+#include <zephyr/ztest.h>
 #include <custom_lib/custom_lib.h>
 
 static void test_get_value(void)
```

Comment:
tests need to use the new ZTEST API.

Reply:
That sounds like a separate enhancement (beyond the scope of a hotfix).

Reply:
I disagree here. example-application needs to show best practices, not outdated ones.

Reply:
indeed

Reply:
this would have not been accepted if PR had been opened like this today, so it has to be fixed.

Reply:
if not we can always revert and ask the author to re-open the PR with updated code. But -1 on keeping this because of an accidental merge.

Reply:
If you will review and merge the PR in a timely manner I'll gladly re-open the PR with updated code.